ground/src/main/java/com/google/android/ground/ui/common/EphemeralPopups.kt

```diff
@@ -16,25 +16,74 @@
 package com.google.android.ground.ui.common
 
 import android.app.Application
+import android.view.View
 import android.widget.Toast
 import androidx.annotation.StringRes
 import com.google.android.ground.R
+import com.google.android.material.snackbar.Snackbar
 import javax.inject.Inject
 import javax.inject.Singleton
 
 /** Displays short-lived messages such as toasts that are shown over other UI elements. */
 @Singleton
-class EphemeralPopups @Inject constructor(private val context: Application) {
+data class EphemeralPopups @Inject constructor(private val context: Application) {
```

Comment:
I think `data class` here isn't really true to the spirit of what data classes are intended for.

Reply:
I agree, but detekt was forcing me to do it :(

Reply:
I'll see if/how I can surppress it here.